LondonBritishLibrary/orient/BLorient2083.xml

```diff
@@ -0,0 +1,2474 @@
+                                            </msItem>
+                            </msItem>
+                      </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">218</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>350</height>
+                                        <width>307</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="22"/>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Neat gʷǝlḥ script, decreasing in height from 6 mm at the beginning to about 4 mm at the end of the manuscript.</desc>
+                                <date notBefore="1694" notAfter="1706" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="137r" to="177r"/>
+                                    <desc type="Comment">Short notes summarizing the main events of the Passion in the upper margins of the 
+                                        readings for Good Friday.</desc>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus from="218ra" to="218vb"/>
+                                    <desc type="Supplication">Benediction</desc>
+                                    <q xml:lang="gez"><title>ጸሎተ፡ ቡራኬ፡</title> <gap reason="ellipsis"/></q>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#1ra"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="de">Gebra Hemamat angekauft fuer 16 Maria Theresia Thaler 
+                                        <date when="1870-08-08">d. 8 August 1870</date>. 
+                                        <persName role="owner" ref="PRS14422FMayer">F. Mayer</persName></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#28va"/>
```

Comment:
From the catalogue it looks like only catalogue extras numbers 1 and 4 are additiones, the rest are colophon and supplications.  What Strelcyn has under no. 5 is a summary of information from supplications that shouls go under `origin`, not an extra

Reply:
Shall I write into `<origin></origin>` or into `<origDate notBefore="1694" notAfter="1706"></origDate>`?

Reply:
you can put it into `origDate`

Reply:
supplications and colophons should be in most cases (unless these are clear additions) encoded with the relevant `msItem`s https://github.com/BetaMasaheft/Documentation/issues/1003
https://github.com/BetaMasaheft/Documentation/issues/895